README.md

```diff
@@ -68,7 +68,7 @@ libraryDependencies += "io.higherkindness" %% "mu-rpc-client-netty"      % "0.15
 libraryDependencies += "io.higherkindness" %% "mu-rpc-client-okhttp"     % "0.15.1"
```

Comment:
Does this artifact keep the `mu-rpc` prefix?

Reply:
In the common and config modules cases, I think it could be agnostic about rpc or http.